mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -18,11 +18,13 @@ pipeline {
                 sh '''
                     # install pkg dependencies
                     sudo apt update
-                    sudo apt install -y ninja-build wget git python cmake
+                    sudo apt install -y ninja-build wget git python cmake python3-pip
+                    pip3 install pathspec unidiff
                 '''
+                sh "PATH=$PATH:/opt/rocm/llvm/bin/"
```

Comment:
Why is this necessary?

Reply:
git-clang-format needs this path to find clang-format.

Reply:
Could you add a comment and be explicit about this?